demo/admin/src/common/EditPageNode.tsx

```diff
@@ -4,6 +4,7 @@ import { createEditPageNode } from "@comet/cms-admin";
 import { MenuItem } from "@mui/material";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
+export { GQLPageTreeNodeAdditionalFieldsFragment } from "./EditPageNode.generated"; //re-export
```

Comment:
Target generated file has a wrong import (at least on my tests):
`import * as Types from '../src/graphql.generated';`
It should be
`import * as Types from '../../src/graphql.generated';`

Reply:
This is still wrong...

Reply:
somehow forgot to git add a file :/